ciongke/Makefile

```diff
@@ -32,10 +32,13 @@ JENKINS_ADDRESS_FILE = ${HOME}/jenkins-address
 # Useful rules:
 # - create-cluster turns up a cluster then prints out the webhook address.
 # - update-cluster pushes new image versions and updates the deployment.
+# - update-jobs updates the configmap defining the Jenkins jobs.
 # - hook-image builds and pushes the hook image.
 # - hook-deployment updates the deployment.
 # - hook-service create the hook service.
-# - test-pr-image builds and pushes the test-pr image.
+# - line-image builds and pushes the line image.
```

Comment:
the name is clever, but I feel like it doesn't do as good a job at being obvious like "hook" and "sinker" do.